app/src/main/java/org/mozilla/fenix/share/SaveToPDFMiddleware.kt

```diff
@@ -37,7 +40,10 @@ class SaveToPDFMiddleware(
             // See https://github.com/mozilla-mobile/fenix/issues/27649 for more details,
             // why a Toast is used here.
             GlobalScope.launch(Dispatchers.Main) {
-                Toast.makeText(context, R.string.unable_to_save_to_pdf_error, LENGTH_LONG).show()
+                ThreadUtils.runOnUiThread {
```

Comment:
Had to use this way of executing on main thread, otherwise I would get:
> Exception in thread "BrowserStore-thread-1 @coroutine#7" java.lang.RuntimeException: Can't toast on a thread that has not called Looper.prepare()

Reply:
I wasn't able to reproduce, which devices are you using to test?

Reply:
I am currently using the emulator. The exception is thrown when running the test created.

Reply:
Which version does the emulator have Android ?

Reply:
Emulator runs on android 13, API level 33

Reply:
Thanks!